specs/AppLifecycle/EnvironmentVariables/AppLifecycle Environment Variables.md

```diff
@@ -530,24 +530,24 @@ private void AppendToUserPathExt()
 
 The new types are defined as follows:
 
-//
-https://github.com/microsoft/ProjectReunion/blob/EVAndPATH/dev/Microsoft.Process.Environment/EnvironmentManager.idl
-
 ```idl
-runtimeclass EnvironmentManager
+namespace Microsoft.Windows.System
```

Comment:
This will produce Microsoft.Windows.System.winmd. You need to add references to that in various places

search the build/ tree for "microsoft.windows.applicationmodel" and you'll find 6 files needing updates

Reply:
Good catch.  Searched for Microsoft.Windows.ApplicationModel.winmd.  Copied each refrence and changed ApplicationModel to System.

Reply:
Wait.  No